codepropertygraph/src/main/scala/io/shiftleft/passes/DiffGraphProtoSerializer.scala

```diff
@@ -150,12 +148,20 @@ class DiffGraphProtoSerializer {
       .setPropertyName(EdgePropertyName.valueOf(propertyKey))
       .build
 
-  private def nodeProperty(key: String, value: Any) = {
-    CpgStruct.Node.Property
-      .newBuilder()
-      .setName(NodePropertyName.valueOf(key))
-      .setValue(protoValue(value))
-      .build()
+  //fixme
+  private def nodeProperty(key: String, value: Any, appliedDiffGraph: AppliedDiffGraph) = {
+    if (key(0).isLower) {
```

Comment:
This disguishment looks fishy. Having a key start with lower case to encode it as CONTAINED_REF is a way to implicit encoding.